### PR TITLE
demo-squeezedet fixed to run default command with all 4 models

### PR DESCRIPTION
**Pull request by Chunosov**

### CODE REVIEW

File: package/demo-squeezedet-patched/patch.linux/demo.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+index 3ddafab..3b4a7de 100644
+--- a/src/demo.py
++++ b/src/demo.py
+@@ -55,7 +55,7 @@ def video_demo():
+   # out = VideoWriter(out_file_name, frameSize=(1242, 375))
+   # out.open()
+ 
+-  assert FLAGS.demo_net == 'squeezeDet' or FLAGS.demo_net == 'squeezeDet+', \
++  assert FLAGS.demo_net == 'squeezeDet' or FLAGS.demo_net == 'squeezeDet+' or FLAGS.demo_net == 'resnet50' or FLAGS.demo_net == 'vgg16', \
+       'Selected nueral net architecture not supported: {}'.format(FLAGS.demo_net)
```
Comment (psyhtest) [inline package/demo-squeezedet-patched/patch.linux/demo.patch]: neural (also elsewhere)

File: program/squeezedet/image_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+from config import *
+from train import _draw_box
+from nets import *
+
+FLAGS = tf.app.flags.FLAGS
+
+tf.app.flags.DEFINE_string('checkpoint', '', """Path to the model parameter file.""")
+tf.app.flags.DEFINE_string('input_file', '', """Input image to be detected.""")
+tf.app.flags.DEFINE_string('demo_net', 'squeezeDet', """Neural net architecture.""")
+tf.app.flags.DEFINE_integer('batch_size', 5, """Batch size.""")
```
Comment (psyhtest) [inline program/squeezedet/image_benchmark.py]: Make it 1 by default?

File: program/squeezedet/image_benchmark.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```
Comment (psyhtest) [inline program/squeezedet/image_benchmark.sh]: Can you move this into metadata please, i.e. make a command key and define the env variables for it appropriately e.g. [for individual command](https://github.com/dividiti/ck-tensorrt/blob/master/program/tensorrt-test/.cm/meta.json#L155) and [for all commands](https://github.com/dividiti/ck-tensorrt/blob/master/program/tensorrt-test/.cm/meta.json#L168). You wouldn't need separate scripts for Windows and Linux then?

File: program/squeezedet/image_benchmark.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+export PYTHONPATH="$PYTHONPATH:$CK_ENV_DEMO_SQUEEZEDET_SRC"
+export TF_CPP_MIN_LOG_LEVEL=3 # supress TF debug output
+
+CHECKPOINT=$CK_ENV_MODEL_SQUEEZEDET_MODEL
+NET=$CK_ENV_MODEL_SQUEEZEDET_ID
+INPUT=$CK_ENV_DEMO_SQUEEZEDET_ROOT/data/sample.png
```
Comment (psyhtest) [inline program/squeezedet/image_benchmark.sh]: Should take images from a dataset?

File: package/demo-squeezedet-patched/patch.linux/demo.patch (hunk 1 above)
Comment (Chunosov) [reply package/demo-squeezedet-patched/patch.linux/demo.patch]: Fixed.
But it is the line from original code and changed line is only one before. Should we apply patches for all founded typos? :)

File: program/squeezedet/image_benchmark.py (hunk 2 above)
Comment (Chunosov) [reply program/squeezedet/image_benchmark.py]: Done

File: program/squeezedet/image_benchmark.sh (hunk 3 above)
Comment (Chunosov) [reply program/squeezedet/image_benchmark.sh]: Moved to meta, but I can't define vars for individual command in `run_set_env2`. Var defined here is not passed as program parameter. So I've stayed them global.

File: program/squeezedet/image_benchmark.sh (hunk 4 above)
Comment (Chunosov) [reply program/squeezedet/image_benchmark.sh]: I've just reused approach of `default` command here. Taking a dataset path we could process 'real' batches with different images. But there are several questions:
- Which tags should we use for search for appropriate dataset? 
- What should we do if specified dataset does not contain `batch_size * batch_count` images?

Comment (Chunosov) [reply program/squeezedet/image_benchmark.sh]: I've done it as in `use_continuous` mode

**Pull request merged**